src/client/fontengine.cpp

```diff
@@ -248,18 +261,6 @@ gui::IGUIFont *FontEngine::initFont(const FontSpec &spec)
 	g_settings->getU16NoEx(setting_prefix + "font_shadow_alpha",
 			font_shadow_alpha);
 
-	std::string path_setting;
```

Comment:
Note to reviewers: I moved this code block down to where `media_name` is actually used.